### ISSUE TITLE
GitHub link colors are not accessible in dark mode

### ISSUE THREAD

**Issue by felixbade** [felixbade/chatgpt-web-ui#34]
Contrast is 3.83, which is less than WCAG minimum 4.5.
<img width="612" alt="Screenshot 2023-03-04 at 0 58 16" src="https://user-images.githubusercontent.com/6554998/222849949-f403a1c8-38ca-43b1-9cd7-a0e2955936ff.png">

**Comment by felixbade** [comment]
Fixed in 6b2f239
